presto-orc/src/main/java/com/facebook/presto/orc/writer/TimestampColumnWriter.java

```diff
@@ -57,7 +57,7 @@
         implements ColumnWriter
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(TimestampColumnWriter.class).instanceSize();
-    private static final int MILLIS_PER_SECOND = 1000;
+    private static final long MILLIS_PER_SECOND = 1000;
```

Comment:
Why is this change? Could you be compiling with the "wrong" version of JDK?

Reply:
it was some refactor i was trying, will revert